### ISSUE TITLE
Active keyword for keyboard hotkeys

### ISSUE THREAD

**Issue by ahuang11** [pyviz/panel#547]
```
import panel as pn
pn.extension()

pn.widgets.Player()
```
Is there a way to make the slider react to my keyboard's arrow keys without having to click on it first?

e.g. if you visit
https://www.tropicaltidbits.com/analysis/models/?model=gfs&region=us&pkg=mslp_pcpn&runtime=2019072412&fh=6
you can immediately use arrow keys

**Comment by jsignell** [comment]
I don't think it is clear what should happen in the case of multiple widgets, so this doesn't seem like something that will be implemented.

**Comment by ahuang11** [comment]
In most cases, multiple widgets will default to False unless user sets it to True?

`keyboard=False`

**Comment by jsignell** [comment]
Ok that seems like a reasonable proposal. Please feel free to open a PR :)
